packages/openmrs/src/Adaptor.js

```diff
@@ -60,23 +36,7 @@ async function login(state) {
  * @returns {Operation}
  */
 export function getPatient(uuid, callback = false) {
-  return state => {
-    Log.info(`Searching for patient with uuid: ${uuid}`);
```

Comment:
I like this level of logging. My sense is that if this was here before, we should keep it. Before `return sendRequest` you could add:

```js
console.log(`Searching for patient with uuid: ${uuid}`);
```